.github/workflows/ci-model-regression-on-schedule.yml

```diff
@@ -12,6 +12,7 @@ env:
   TF_FORCE_GPU_ALLOW_GROWTH: true
   GITHUB_ISSUE_LABELS: '["type:bug :bug:", "tool:model-regression-tests"]'
   PERFORMANCE_DROP_THRESHOLD: -0.05
+  NVML_INTERVAL_IN_SEC: 1
```

Comment:
Do you need to collect it every second?

Reply:
Depends on the use case. One user reported this would be helpful, but other users might not need this granularity.
Do you think a short default interval would have disadvantages?

Reply:
Rather I was wondering what is the use case that the default doesn't work, but if it's going to help then I'm ok with it.